stress/shell/src/main/java/alluxio/stress/cli/UfsIOBench.java

```diff
@@ -53,6 +53,11 @@
 
   private final UUID mTaskId = UUID.randomUUID();
 
+  @Override
+  public String getBenchDescription() {
+    return "";
```

Comment:
Can you help add a TODO(jiacheng) add description here 
FYI @jiacheliu3

Reply:
roger that, i'll work on the TODO after this PR is in